viewing_party/party.py

```diff
@@ -1,23 +1,168 @@
+import copy
 # ------------- WAVE 1 --------------------
+movies = {}
 
 def create_movie(title, genre, rating):
-    pass
+    if title != None and genre != None and rating != None:
+        movies.update({"genre": genre, "rating": rating, "title": title})
+        return movies
+    
+    else:
+        return None
 
+def add_to_watched(user_data, movie):
+    user_data.update({"watched": [movie]})
+    return user_data

```

Comment:
The update call here will replace the entire `watched` list with a new list that only contains `movie`. If we had users, they would see their entire watch history cleared each time they finish a new movie! In this case we should `.append()` our `movie` object to our list to preserve whatever data might already be in our `watched` list.